distribution/lib/Standard/Base/0.0.0-dev/src/Runtime/Debug.enso

```diff
@@ -0,0 +1,36 @@
+## Debug utilities.
+## A type on which debugging functionality is exposed.
+@Builtin_Type
+type Debug
```

Comment:
```suggestion
## Debug utilities.

## A type on which debugging functionality is exposed.
@Builtin_Type
type Debug
```

Looks like missing newline.

Again, is this type that has no methods definitely needed?